sdk/typescript/src/rpc/connection.ts

```diff
@@ -43,3 +43,8 @@ export const testnetConnection = new Connection({
   fullnode: 'https://fullnode.testnet.sui.io:443/',
   faucet: 'https://faucet.testnet.sui.io/gas',
 });
+
+export const mainnetConnection = new Connection({
+  fullnode: 'https://fullnode.mainnet.sui.io:443/',
+  faucet: '',
```

Comment:
could we leave `faucet` undefined here instead of specifying an empty string? that'll make it slightly more explicit that there's no faucet for mainnet